components/definitions/newTab.d.ts

```diff
@@ -281,6 +281,7 @@ declare namespace NewTab {
   export interface RewardsParameters {
     rate: number
     monthlyTipChoices: number[]
+    payoutStatus: any
```

Comment:
This is subtle - we don't want to use `any` here, but since this stuff is stored in `localStorage` we can't assume that it will be not `undefined` after upgrade. On option would be to make it optional, and let code that uses it deal with the `undefined` case.

```
payoutStatus?: Record<string, string>
```

Another option would be to add a "migration" to `new_tab_storage.ts`.

Reply:
Good catch, fixed.